thefuck/ui.py

```diff
@@ -84,6 +87,20 @@ def select_command(corrected_commands):
         if action == const.ACTION_SELECT:
             sys.stderr.write('\n')
             return selector.value
+        elif action == const.ACTION_TO_BUFFER:
+            sys.stderr.write('\n')
+            # use a perl trick to pipe selection to terminal input buffer
```

Comment:
Does this work on all shells? Thinking particularly powershell for windows. This might want to be a per shell enabled feature?